src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateFactory.java

```diff
@@ -18,7 +18,10 @@ public static AbstractEmailTemplate get(String bundle, String application) {
                 default:
                     break;
             }
-
+        } else if (bundle.toLowerCase().equals("openshift")) {
+            if (application.toLowerCase().equals("advisor")) {
```

Comment:
@epapbak Could you take a look? Is the called app "advisor" in the bundle "openshift"? 

// cc: @theute

Reply:
Correct

Reply:
Thanks guys! Sorry I didn't see this PR until today